solutions/security/detect-and-alert/suppress-detection-alerts.md

```diff
@@ -13,74 +13,60 @@ products:
-    * **Indicator match, event correlation (non-sequence queries only), new terms, {{esql}}, or {{ml}} rules:** Alerts with the specified field name and identical array values are grouped together. For example, if you suppress alerts by `destination.ip` of `[127.0.0.1, 127.0.0.2, 127.0.0.3]`, alerts with the entire array are grouped and only one alert is created for the group.
-    * **Event correlation (sequence queries only) rules:** If the specified field contains an array of values, suppression only happens if the field’s values are an exact match and in the same order. For example, if you specify the field `myips` and one sequence alert has [1.1.1.1, 0.0.0.0] and another sequence alert has [1.1.1.1, 192.168.0.1], neither of those alerts will be suppressed, despite sharing an array element.
+    * **All rule types except the threshold rule:** In **Suppress alerts by**, enter 1-3 field names to group events by the fields' values.
+    * **Threshold rule only:** In **Group by**, enter up to 3 field names to group events by the fields' values, or leave the setting empty to group all qualifying events together. 
 
-    ::::
 
-2. If available, select how often to create alerts for duplicate events:
+    ::::{tip}
+    
+    Refer to [Suppression for fields with an array of values](/solutions/security/detect-and-alert/suppress-detection-alerts.md#security-alert-suppression-fields-with-multiple-values) learn how fields with multiple values are handled.
 
-    ::::{note}
-    Both options are available for custom query, indicator match, event correlation, new terms, {{esql}}, and {{ml}} rules. Threshold rules only have the **Per time period** option.
     ::::
 
 
+2. Choose how often to create alerts for qualifying events:
+
     * **Per rule execution**: Create an alert each time the rule runs and an event meets its criteria.
-    * **Per time period**: Create one alert for all qualifying events that occur within a specified time window, beginning from when an event first meets the rule criteria and creates the alert.
+    * **Per time period**: Create one alert for all qualifying events that occur within a specified time window, beginning from when an event first meets the rule criteria and creates the alert. This is the only option available when configuring alert suppression for threshold rules.
 
         For example, if a rule runs every 5 minutes but you don’t need alerts that frequently, you can set the suppression time period to a longer time, such as 1 hour. If the rule meets its criteria, it creates an alert at that time, and for the next hour, it’ll suppress any subsequent qualifying events.
 
         :::{image} /solutions/images/security-alert-suppression-options.png
```

Comment:
Note to self: This image might need a border. Not sure why the style doesn't match the other screenshots on the page.

Reply:
Fixed in [8edf3d9](https://github.com/elastic/docs-content/pull/2735/commits/8edf3d906274451ba104a1a3f0c562117d9c99e4).